client/js/models/archive.js

```diff
@@ -79,7 +80,6 @@ module.exports = {
   effects: {
     new: function (data, state, send, done) {
       drive = getDrive()
-      var _fs = {}
```

Comment:
aye. in this line, though, i think that _fs will have to exist somewhere so that the closure in file: can access it, right?  where does the variable go after choo calls `send`? https://github.com/datproject/dat.land/blob/a714d131fdbc77989112cf57042ca3efef95693f/client/js/models/archive.js#L107